framework/src/main/kotlin/br/com/zup/beagle/platform/BeaglePlatform.kt

```diff
@@ -46,12 +46,12 @@ enum class BeaglePlatform {
 
     fun isMobilePlatform() = this == MOBILE || this == ANDROID || this == IOS
 
-    fun allowToSendComponentToPlatform(beaglePlatform: BeaglePlatform) =
+    fun allowToSendComponentToPlatform(beaglePlatform:                                                                              BeaglePlatform) =
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/852ca04da8603a3a73e6cad9cefeef6a5add212d/framework/src/main/kotlin/br/com/zup/beagle/platform/BeaglePlatform.kt#L49"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>